src/foam/nanos/dig/drivers/DigFormatDriver.js

```diff
@@ -200,7 +201,7 @@ foam.CLASS({
       PrintWriter out = x.get(PrintWriter.class);
       out.println();
       out.flush();
-      getLogger().debug("select.success");
+      getLogger().debug("select.success", daoName, id);
```

Comment:
I think you should only include the 'id' if the check from line 171 passes (SafetyUtil.isEmpty(id)), or not at all, since I don't think 'id' is used much for selects (but I could be wrong).
Also, given that these log messages don't contain any extra information, would PM's be even better? Or do you need to see where they are interleaved in with the other log messages?